config/prow/cluster/ingress-nginx/ingress-nginx-deployment.yaml

```diff
@@ -788,16 +789,16 @@ subjects:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
-  name: ingress-nginx-admission
```

Comment:
This whitespace change seems to be unnecessary.

Reply:
That's true 😅 
The file is generated from the helm chart. The additional space comes from [here](https://github.com/kubernetes/ingress-nginx/blob/5583f90c7f3525b6267747d6b07541c01218a9ea/charts/ingress-nginx/templates/admission-webhooks/job-patch/role.yaml#L5C8-L5C8) and was introduced lately with [this commit](https://github.com/kubernetes/ingress-nginx/commit/7ce6cc88d8501628fd0fe1fb882579ad7d2a95fe#diff-3d10afa8f48a66b5503389d8b56e9d9741898bc277d6e0c21efa43cfe3c2bf62).

The additional space seems to be already removed on main, so I removed it here too 😄